<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>telaLoginColaborador</class>
 <widget class="QDialog" name="telaLoginColaborador">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>204</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#telaLoginColaborador{
	background-color: #cffdff;
}

QPushButton{
	background-color: #afdcde;
	border-radius: 14px;
	border: 2px solid #3d5557;
	color: #3d5557;
	font-size: 16px;
}

QPushButton::hover{
	background-color: #b9faf4;
}

QLabel{
	color: #3d5557;
	font-weight:bold;
}

QLineEdit {
	border-radius: 4px;
	border: 1px solid #3d5557
}
</string>
  </property>
  <widget class="QLabel" name="label_infoinvalida">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Username ou senha inv√°lidos</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>441</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>4</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Username:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_username"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Senha:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_senha">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btn_login">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">#btn_login{
border-radius: 14px;
}</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
